src/mic/__main__.py

```diff
@@ -250,32 +268,41 @@ def step4(mic_file, configuration_files):
     """
     THIS IS STEP IS OPTIONAL
```

Comment:
We have agreed that this should not be like this. Instead, it should be: 

"If your model does not use configuration files, you can skip this step"